packages/apps/reputation-oracle/server/src/modules/encryption/encryption.module.ts

```diff
@@ -0,0 +1,12 @@
+import { Module } from '@nestjs/common';
+
+import { Web3Module } from '../web3/web3.module';
+
+import { PgpEncryptionService } from './pgp-encryption.service';
+
+@Module({
+  imports: [Web3Module],
+  providers: [PgpEncryptionService],
+  exports: [PgpEncryptionService],
+})
+export class EncryptionModuleModule {}
```

Comment:
```suggestion
export class EncryptionModule {}
```

Reply:
Fixed in https://github.com/humanprotocol/human-protocol/pull/3249/commits/8a10ca865fd8b839897f54d3d5e601c0550c1f81